Packages/Signer/Tests/SignerTests/SwapSignerTests.swift

```diff
@@ -201,6 +207,71 @@ struct SwapSignerTests {
         #expect(captured.destinationAddress != swapData.data.to)
     }
 
+    @Test
+    func trc20TransferSwapUsesTokenTransferAndParsesBase58Destination() throws {
+        let fromAsset = Asset(
```

Comment:
```suggestion
        let fromAsset = Asset.mock()
```

use this to avoid an extra data to define